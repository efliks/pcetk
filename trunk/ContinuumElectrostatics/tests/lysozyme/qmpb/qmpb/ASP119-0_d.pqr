../0/site_ASP_119_d.pqr