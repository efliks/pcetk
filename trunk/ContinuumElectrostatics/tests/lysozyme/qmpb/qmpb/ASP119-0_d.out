Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP119-0_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (-7.91, 35.596, 8.212) 101 1
Finer level:    (-7.91, 35.596, 8.212) 101 0.5
Finest level:   (-7.91, 35.596, 8.212) 101 0.25


Self energy of ASP119-0_d in three dielectric environment = 383.08
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP119-0_d with background charge set in three dielectric environment = -10.5452
(probably in kcal/mole)



Interaction energies of ASP119-0_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.0101991
0   1 : -0.000110149
1   0 : -0.000880343
1   1 : 0.0178134
2   0 : -0.0450228
2   1 : 0.000279605
3   0 : -0.0114439
3   1 : -0.00234879
3   2 : 0.00133953
3   3 : 0.00958542
4   0 : 0.00804302
4   1 : 0.0887155
5   0 : -0.000222826
5   1 : 0.0118281
6   0 : 0.00434038
6   1 : 0.0624995
7   0 : -0.101699
7   1 : -0.000399245
8   0 : -0.000578855
8   1 : 0.0419325
9   0 : 0.000336734
9   1 : 0.00563955
10   0 : -0.000199525
10   1 : 0.0148526
11   0 : -6.9654e-05
11   1 : 0.0040308
12   0 : -2.02011e-05
12   1 : 0.00262859
13   0 : 0.000220441
13   1 : 0.00520191
14   0 : -0.0101409
14   1 : -0.00042774
15   0 : -0.00275315
15   1 : -4.38663e-05
16   0 : -0.000163446
16   1 : 0.00635937
17   0 : -0.0586992
17   1 : -0.00122712
18   0 : 525.261
18   1 : 766.16
Total runtime: 7.14 s user 0.06 s system 
